### ISSUE TITLE
Using $save $update and $delete shortcuts

### ISSUE THREAD

**Issue by nicooprat** [CodeDredd/pinia-orm#1065]
### Describe the feature

Hi there,

In vuex-orm we can use `$` prefixed methods on entities. It looks like it doesn't exist in pinia-orm. It's not mandatory as it's possible todo it differently, but it makes the migration a bit longer.

Example: https://vuex-orm.org/guide/data/inserting-and-updating.html#save-method

```js
const user = new User()
user.name = 'John Doe'
user.$save()
```

Is it removed on purpose? Do you plan to add it? Or is there an existing alternative I missed?

Thanks

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Check existing [discussions](https://github.com/CodeDredd/pinia-orm/discussions) and [issues](https://github.com/CodeDredd/pinia-orm/issues).

**Comment by CodeDredd** [comment]
Hey @nicooprat,

it is removed on purpose. If you look at `vuex-orm-next` it is the same. Vuex ORM Next started to move from Model Actions to repository pattern. So logic is now more kept in the repository and the model is just a transformation layer.
You can compare it to moving from Eloquent to doctrine. ^^
It's not planned to have these methods on model level.
I know it makes migration harder. I think some1 here started wrting a `bridge` so that transistion from vuex-orm to pinia-orm is a bit more easier. I also thought about making this kind of bridge offical to help others. But haven't the time yet for it.

As always contriubution (even just telling what the most pains are) are very welcome. Orgiving me some cups of ☕ helps too ❤️ hehehe

**Comment by nicooprat** [comment]
Thanks for the clarification!

So there's only the `useRepo(User).save(...)` method, where it creates a new record when the `id` field is missing, or updates it if `id` is specified?

**Comment by CodeDredd** [comment]
Correct. With https://github.com/CodeDredd/pinia-orm/pull/790 saving gets a bit more consistence to help debugging. But this will come with v2.
